release-notes-3.0.0.md

```diff
@@ -11,3 +11,6 @@ ${version-number}
 #### New Features
 
 #### Known Issues
+
+- [SCMOD-6618](https://portal.digitalsafe.net/browse/SCMOD-6618): Trying to updating a job could incorrectly succeed  
```

Comment:
This should be in a "Bug Fixes" section rather than in a "Known Issues" section.  Known issues is for issues which havn't been fixed.  See [SCMOD-6383](https://portal.digitalsafe.net/browse/SCMOD-6383) for some advice around the release notes sections.  I think I'd consider this to be a trivial fix and leave it out of the release notes altogether.